README.md

```diff
@@ -4,111 +4,136 @@ An API for validating survey schemas.
 
 ## Setup
 
-It is recommended that you use [Pyenv](https://github.com/pyenv/pyenv) to manage your Python installations.
+In order to run locally you'll need Node.js, poetry and pyenv installed.
 
-### Install Poetry
+### Install NVM and pyenv
+
+NVM and pyenv will manage your versions of Node and Python.
+``` shell
+brew install nvm pyenv
```

Comment:
No other mention of pyenv later despite we say to use `nvm use` command for Node.